cdk/lib/newswires.ts

```diff
@@ -153,31 +155,35 @@ export class Newswires extends GuStack {
 			dimensions: { sourceFeed: '$.message.sourceFeed' },
 		});
 
-		const scheduledCleanupLambda = new GuScheduledLambda(this, `ScheduledCleanupLambda-${this.stage}`, {
```

Comment:
this block is just auto formatting I believe; will look into why this isn't flagged by CI..

Reply:
ah, we're not running prettier as part of eslint here. I'll take a look at fixing that today, but in the meantime I think that this change is in line with prettier config so I'm inclined to leave it here for now unless there are objections 👍